command/vsm_stats.go

```diff
@@ -427,17 +428,17 @@ func StatsOutput(c *VsmStatsCommand, annotations *Annotations, args []string, st
 
 	} else {
 
-		// Printing in tabular form
-		//	fmt.Printf("%+v\n\n", annotation)
-		data, err := json.MarshalIndent(annotation, "", "\t")
-
+		// Printing using template
+		tmpl, err := template.New("test").Parse("IQN     : {{.IQN}}\nVolume  : {{.Volume}}\nPortal  : {{.Portal}}\nSize    : {{.Size}}")
```

Comment:
Use typed string constant for the string used in template.
Eventually this typed string will be used to define many more templates.